src/rdo.rs

```diff
@@ -263,10 +295,11 @@ pub fn rdo_mode_decision(
 
           let mut wr: &mut dyn Writer = &mut WriterCounter::new();
           let tell = wr.tell_frac();
-
+          if skip { tx_type = TxType::DCT_DCT; };
```

Comment:
tx_type should be changed only for the current iteration, not all the following ones as well

Reply:
Oops, correct. It should be the same for whole chroma loop.